tfhe/src/core_crypto/algorithms/test/noise_distribution/lwe_encryption_noise.rs

```diff
@@ -93,7 +93,7 @@ fn lwe_compact_public_key_encryption_expected_variance(
     lwe_dimension: LweDimension,
 ) -> Variance {
     let input_variance = input_noise.get_variance();
-    Variance(input_variance * (lwe_dimension.to_lwe_size().0 as f64))
+    Variance(input_variance.0 * (lwe_dimension.to_lwe_size().0 as f64))
```

Comment:
We could have a `Mul<Rhs = usize>` for 'Variance'

Reply:
I did not want to update the usability, just that we return strong types, this can be done in a follow up

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/841